doc/api/configuration.md

```diff
@@ -198,31 +198,17 @@ To unset:
 
 ## `feedback`
 
-Default: `True`
-
-Print number of rows affected by DML.
-
-```{code-cell} ipython3
-%config SqlMagic.feedback = True
+```{versionchanged} 0.10
+`feedback` takes values `0`, `2`, and `2` instead of `True`/`False`
 ```
 
-```{code-cell} ipython3
-%%sql
-CREATE TABLE my_points (x, y);
-INSERT INTO my_points VALUES (0, 0);
-INSERT INTO my_points VALUES (1, 1);
-```
+Default: `1`
 
-```{code-cell} ipython3
-%config SqlMagic.feedback = False
-```
+Control the quantity of feedback displayed when performing certain operations:
 
-```{code-cell} ipython3
-%%sql
-CREATE TABLE more_points (x, y);
-INSERT INTO more_points VALUES (0, 0);
-INSERT INTO more_points VALUES (1, 1);
-```
+- `0`: No feedback
+- `1`: Normal feedback (default)
+- `2`: All feedback
```

Comment:
What does `all feedback` include? From the below screenshot the difference between `1` and `2` seems `ResultSet: to convert to pandas, call .DataFrame() or to polars, call .PolarsDataFrame()`.

<img width="783" alt="Screenshot 2023-08-14 at 7 11 43 PM" src="https://github.com/ploomber/jupysql/assets/29096135/b7dd01d0-0991-4941-a0c8-7680d240f51f">

<img width="667" alt="Screenshot 2023-08-14 at 7 11 50 PM" src="https://github.com/ploomber/jupysql/assets/29096135/b76f8c89-88b4-45ab-a4e4-729184ce6821">

Reply:
fixed https://github.com/ploomber/jupysql/pull/798/commits/1ab93c2432686d37782d39eebe364876090ba249